core/templates/dev/head/domain/utilities/language-util.service.spec.ts

```diff
@@ -16,98 +16,89 @@
-      }, {
-        code: 'ar',
-        description: 'العربية (Arabic)'
-      }, {
-        code: 'bg',
-        description: 'български (Bulgarian)'
-      }];
-      mockAutogeneratedAudioLanguages = [{
-        id: 'en-auto',
-        description: 'English (auto)',
-        exploration_language: 'en',
-        speech_synthesis_code: 'en-GB',
-        speech_synthesis_code_mobile: 'en_US'
-      }];
-      var mockSupportedAudioLanguages = [{
-        id: 'en',
-        description: 'English',
-        relatedLanguages: ['en']
-      }, {
-        id: 'hi-en',
-        description: 'Hinglish',
-        relatedLanguages: ['hi', 'en']
-      }, {
-        id: 'es',
-        description: 'Spanish',
-        relatedLanguages: ['es']
-      }];
-      $provide.constant('SUPPORTED_CONTENT_LANGUAGES',
-        mockSupportedContentLanguages);
-      $provide.constant('AUTOGENERATED_AUDIO_LANGUAGES',
-        mockAutogeneratedAudioLanguages);
-      $provide.constant('SUPPORTED_AUDIO_LANGUAGES',
-        mockSupportedAudioLanguages);
-      // Chrome loads voices asynchronously.
-      // https://stackoverflow.com/questions/21513706/
-      window.speechSynthesis.onvoiceschanged = function() {
-        $provide.value('BrowserCheckerService'), {
-          supportsSpeechSynthesis: function() {
-            return true;
-          }
-        };
-      };
-    });
+  var SUPPORTED_CONTENT_LANGUAGES = CONSTANTS.SUPPORTED_CONTENT_LANGUAGES;
+  var AUTOGENERATED_AUDIO_LANGUAGES = CONSTANTS.AUTOGENERATED_AUDIO_LANGUAGES;
+  var SUPPORTED_AUDIO_LANGUAGES = CONSTANTS.SUPPORTED_AUDIO_LANGUAGES;
+
+  beforeEach(() => {
+    var mockSupportedContentLanguages = [{
+      code: 'en',
+      description: 'English'
+    }, {
+      code: 'ar',
+      description: 'العربية (Arabic)'
+    }, {
+      code: 'bg',
+      description: 'български (Bulgarian)'
+    }];
+    mockAutogeneratedAudioLanguages = [{
+      id: 'en-auto',
+      description: 'English (auto)',
+      exploration_language: 'en',
+      speech_synthesis_code: 'en-GB',
+      speech_synthesis_code_mobile: 'en_US'
+    }];
+    var mockSupportedAudioLanguages = [{
+      id: 'en',
+      description: 'English',
+      relatedLanguages: ['en']
+    }, {
+      id: 'hi-en',
+      description: 'Hinglish',
+      relatedLanguages: ['hi', 'en']
+    }, {
+      id: 'es',
+      description: 'Spanish',
+      relatedLanguages: ['es']
+    }];
+
+    CONSTANTS.SUPPORTED_CONTENT_LANGUAGES = mockSupportedContentLanguages;
+    CONSTANTS.AUTOGENERATED_AUDIO_LANGUAGES = mockAutogeneratedAudioLanguages;
+    CONSTANTS.SUPPORTED_AUDIO_LANGUAGES = mockSupportedAudioLanguages;
+    // Chrome loads voices asynchronously.
+    // https://stackowindow.speechSynthesis.onvoiceschanged = function() {
+    //   $provide.value('BrowserCheckerService'), {
+    //     supportsSpeechSynthesis: function() {
+    //       return true;
+    //     }
+    //   };
+    // };verflow.com/questions/21513706/
+    //
```

Comment:
What happened here?

Reply:
I couldn't find the use of this variable anywhere and hence I removed this.

Reply:
If this isn't used, remove it instead of commenting it out.